icd_windows_hkr.c

```diff
@@ -115,7 +96,7 @@ static bool ReadOpenCLKey(DEVINST dnDevNode)
     {
         result = RegQueryValueExA(
             hkey,
-            GetOpenCLRegKeyName(),
+            OPENCL_REG_SUB_KEY,
```

Comment:
We should use the GetOpenCLRegKeyName() here and also in icd_windows_hkr.c line 139, I missed that line on my previous patch.

Reply:
@saleelk,
Are we fixing this as part of this commit?